data/korath/korath missions.txt

```diff
@@ -257,3 +257,54 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+				"reputation: Korath" >= -100
+			and
+				random < 10
+				"reputation: Korath" < -100
+				"reputation: Korath" >= -1000
+			and
+				random < 25
+				"reputation: Korath" < -1000
+				"reputation: Korath" >= -10000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+				"reputation: Korath (Civilian)" >= -300
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -300
+				"reputation: Korath (Civilian)" >= -3000
+			and
+				random < 50
+				"reputation: Korath (Civilian)" < -3000
+	source
+		government Korath "Korath (Civilian)"
+		system Alcyone Almach "Alpha Hydri" Atik Durax Misam Polaris Sheratan
+	on offer
+		"korath: boarding response count" ++
+		conversation
+			"An angry hiss comes over your radio, and you see a strange pulse along the hull of the Korath vessel. Your ship detects a powerful hyperwave emission followed by several capital ships jumping in. The pulse throws your ship off of theirs, but it appears you could board them again. You had best finish your business here before you are reduced to a ball of superheated plasma."
```

Comment:
What is a "hyperwave"?

Reply:
It is the human word for their half-understanding of the concept of waves traveling along the weft. Just like radios use radio waves, hyperspace relays use hyperwaves. Naturally, since this is the standard form of interstellar communication, your ship knows how to pick up hyperwaves and inform you of them.

Seriously. They covered this in pilot training. Why did you pay 50,000 credits if you were just going to forget all of this?

Reply:
There are no hyperspace links between the systems where this missino can occur and any reasonable place for these Korath ships to be waiting.
The only possible exceptions are Durax and Alcyone where Atik and/or Misam could potentially serve as staging grounds, but I'm not sure that should be the case.

Reply:
The Korath don't need starlanes to communicate between systems.